src/components/image/image.html

```diff
@@ -0,0 +1,146 @@
+                    focusable="false" height="1em" width="1em" xmlns="http://www.w3.org/2000/svg">
+                    <path
+                        d="M256 160c-52.9 0-96 43.1-96 96s43.1 96 96 96 96-43.1 96-96-43.1-96-96-96zm246.4 80.5l-94.7-47.3 33.5-100.4c4.5-13.6-8.4-26.5-21.9-21.9l-100.4 33.5-47.4-94.8c-6.4-12.8-24.6-12.8-31 0l-47.3 94.7L92.7 70.8c-13.6-4.5-26.5 8.4-21.9 21.9l33.5 100.4-94.7 47.4c-12.8 6.4-12.8 24.6 0 31l94.7 47.3-33.5 100.5c-4.5 13.6 8.4 26.5 21.9 21.9l100.4-33.5 47.3 94.7c6.4 12.8 24.6 12.8 31 0l47.3-94.7 100.4 33.5c13.6 4.5 26.5-8.4 21.9-21.9l-33.5-100.4 94.7-47.3c13-6.5 13-24.7.2-31.1zm-155.9 106c-49.9 49.9-131.1 49.9-181 0-49.9-49.9-49.9-131.1 0-181 49.9-49.9 131.1-49.9 181 0 49.9 49.9 49.9 131.1 0 181z">
+                    </path>
+                </svg>
+            </div>
+        </div>
+    </header>
+
+    <!-- aside start -->
+    <aside>
+        <!-- Getting Started -->
+        <h3>Getting Started</h3>
+        <a href="">Introduction</a>
+        <a href="">Typography</a>
+
+        <!-- components -->
+        <h3>Components</h4>
+            <a href="../alert/alert.html">Alert</a>
+            <a href="../avatar/avatar.html">Avatar</a>
+            <a href="../badge/badge.html">Badge</a>
+            <a href="../button/button.html">Button</a>
+            <a  href="../card/card.html">Card</a>
+            <a class="active-tab" href="">Image</a>
+            <a href="../input/input.html">Input</a>
+            <a href="">Form</a>
+            <a href="">Snackbar</a>
+    </aside>
+
+    <!-- main -->
+    <main>
+        
+        <!-- image round -->
+        <h1>Image</h1>
+        <p class="body-l">
+            
+        </p>
+        <div class="component-desc">
+            <h3>Round Image</h3>
+            <p >Add class <span class="highlight-text">img-round</span><br>
+            Here we used 2 images one with 800*400 and another 400*400</p>
+        </div>
+
+        <div class="component-container">
+            <div class="artsy-img img-round ">
+                <img src="https://picsum.photos/800/400" alt ="random images" loading="lazy" />
+            </div>
+            <div class="artsy-img img-round ">
+                <img src="https://picsum.photos/400/400" alt ="random images" loading="lazy" />
+            </div>
+        </div>
+
+        <!-- image responsive -->
+        <div class="component-desc">
+        <h3>Responsive Image</h3>
+        <p >By default, the images in artsy ui occupy 100% of their containers width and has object-fit: cover.</p>
+        </div>
+
+        <div class="component-container">
+            <div class="artsy-img">
+                
+                <img src="https://picsum.photos/300/200" alt ="random images" loading="lazy" />
+            </div>
+            <div class="artsy-img img-responsive">
+                <img src="https://picsum.photos/300/200" alt ="random images" loading="lazy" />
+            </div>
+            
```

Comment:
The problem of using img-responsive like this is the initial intrinsic dimensions of image is small (300-200) and then when you do width: 100% in css it blurs, that is why srcset is a good option. In srcset attribute you provide img set and browser chooses best image according to available dimensions.

Other option is you use high resolution image as original image and then work with its dimensions.